Limelight/ViewControllers/MainFrameViewController.m

```diff
@@ -472,7 +472,11 @@ - (void) prepareToStreamApp:(TemporaryApp *)app {
     
     // multiController must be set before calling getConnectedGamepadMask
     _streamConfig.multiController = streamSettings.multiController;
-    _streamConfig.gamepadMask = [ControllerSupport getConnectedGamepadMask:_streamConfig];
+    if (!streamSettings.disableTvOSController) {
```

Comment:
Why are you thinking of a separate option for this? TvOS doesn't have on-screen controls, so we should report 0 gamepads unless a real one is connected to the Apple TV

Reply:
Even if no controllers are connected, the connected remote acts as a controller.  I'll see if I can work around that, but it may not be ideal.

Reply:
Yeah, we should exclude micro gamepads (which I thought we did already).

Reply:
i figured what the source of this is -- if no controllers are connected and multi-controller support is disabled, the controller mask will always be set to `0x1`, regardless of the on-screen control setting.  i updated this so that, if running on tvOS, the mask will not be updated (and default to 0).  this also means the micro-controller (siri remote) can also act as a mouse, even with a controller connected to the pc.

i was also able to remove the new setting.